less/navigation.less

```diff
@@ -0,0 +1,34 @@
+header {
+    .flex(column,center,center);
+    .width(1000px,100%);
+    background-color: @hfc;
+    border-bottom: 1px solid @borderc;
+    padding: 10px 0px;
+
+    h1 {
+        font-size: 3rem;
+
+        @media @phone {
+            font-size: 5.5em;
+            }
+        }
+    }
```

Comment:
Your spacing here is a bit off. Consider using extensions such as `Beautify` and `Prettier` to clean up your CSS.